app/common/controllers/intercom.js

```diff
@@ -0,0 +1,28 @@
+module.exports = [
+    '$scope',
+    '$rootScope',
+    'UserEndpoint',
+function (
+    $scope,
+    $rootScope,
+    UserEndpoint
+) {
+    $rootScope.$on('event:authentication:login:succeeded', function () {
```

Comment:
looks like intercom already does some 'Session' tracking itself https://docs.intercom.io/faqs/how-is-a-session-defined